bcipy/signal/model/keyinput.py

```diff
@@ -0,0 +1,34 @@
+from typing import List
+
+
+def compute_keyinput_probs(
```

Comment:
I think this could be more generic than keyinput... 1compute_probabilities_given_inquiry1 or something like that. The filename (keyinput.py) could also be more generic to that point. Whether it's key input or a different kind of evidence this method could be leveraged.